test/runtests.jl

```diff
@@ -3,7 +3,7 @@ if !isdefined(Main, :InterpolationTestUtils)
     @eval using Main.InterpolationTestUtils
 end
 
-using Test, SharedArrays, Random
+using Test, SharedArrays, Random, Images
```

Comment:
`Images` is too big a dependency, just `ImageCore` and/or `ColorVectorSpace` should be sufficient enough. And should also need to modify `Project.toml` to add it as test dependency.

Reply:
Done.